core-ui/src/components/Predefined/List/Pod/PodRestarts.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { StatusBadge } from 'react-shared';
+
+export default function PodRestarts({ statuses }) {
+  const restartCount = statuses.reduce((acc, c) => acc + c.restartCount, 0);
+  const type = restartCount ? 'warning' : 'success';
+
+  const tooltipContent = (() => {
+    if (!restartCount) return null;
+    return (
+      <ul style={{ textAlign: 'left' }}>
+        {statuses.map(s => (
+          <li key={s.name}>{`${s.name}: ${s.restartCount}`}</li>
+        ))}
+      </ul>
+    );
+  })();
```

Comment:
Why did you use a self executing function instead of a normal _Component_?

Reply:
I didn't want to write
```
let tooltipContent = null;
if (restartCount) {
  tooltipContent = 'yadadada'
}
```
nor
```
const getTooltipContent = () => {
	if (!restartCount) return null;
	tooltipContent = ...
}
...tooltipContent={getTooltipContent()}>
```
so I wanted something else.

Yeah, tiny component will be also okay.

Reply:
Or nope, it won't be okay. Let's go with function.